backend/build.gradle

```diff
@@ -26,24 +26,28 @@ repositories {
 }
 
 dependencies {
-	//implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
 	implementation 'org.springframework.boot:spring-boot-starter-data-rest'
 	implementation 'org.springframework.boot:spring-boot-starter-jersey'
 	implementation 'org.springframework.boot:spring-boot-starter-webflux'
+
+	//Security
+	implementation 'org.springframework.boot:spring-boot-starter-security'
+	compile group: 'org.springframework.security', name: 'spring-security-core', version: '5.3.0.RELEASE'
```

Comment:
compile is deprecated afaik
as we use spring dependency management we should not need to specify the version explicitly